src/js/letters/actions/letters.js

```diff
@@ -74,26 +73,34 @@ export function getMailingAddress() {
     apiRequest(
       '/v0/address',
       null,
-      response => {
-        const responseCopy = Object.assign({}, response);
-        const address = Object.assign({}, response.data.attributes.address);
-        // Translate military-only fields into generic ones; we'll translate them back later if necessary
-        if (address.type === ADDRESS_TYPES.military) {
-          address.city = address.militaryPostOfficeTypeCode;
-          address.stateCode = address.militaryStateCode;
-          address.countryName = 'USA';
-          delete address.militaryPostOfficeTypeCode;
-          delete address.militaryStateCode;
+      (response) => {
+        try {
```

Comment:
There are a few things that could go wrong in here, especially if the response doesn't have the shape we think it does (came up a bunch when I was writing tests), so thought it'd be safer to wrap the logic here and reject the promise in the catch.